edgepi-dac.html

```diff
@@ -18,48 +18,46 @@
       return this.name || "dac";
     },
     oneditprepare: function () {
-      // Query html elements
       const configTransportSelect = document.getElementById(
         "node-input-transport"
       );
-      const gainOption1 = document.getElementById("node-input-gain1");
-      const gainOption2 = document.getElementById("node-input-gain2");
+      const gain1 = document.getElementById("node-input-gain1");
+      const gain2 = document.getElementById("node-input-gain2");
       const tcpField = document.querySelector(".form-row.tcp");
       const voltageGain1 = document.querySelector(".form-row.voltageGain1");
       const voltageGain2 = document.querySelector(".form-row.voltageGain2");
 
-      if (this.gainOption == "gain1") {
-        gainOption1.checked = true;
+      if (this.gain == false) {
+        gain1.checked = true;
       } else {
-        gainOption2.checked = true;
+        gain2.checked = true;
       }
 
       function updateEditor() {
         tcpField.style.display =
           configTransportSelect.value === "Network" ? "flex" : "none";
-        const selectedGain = gainOption1.checked ? "gain1" : "gain2";
-        voltageGain1.style.display = selectedGain === "gain1" ? "flex" : "none";
-        voltageGain2.style.display = selectedGain === "gain2" ? "flex" : "none";
+        const selectedGain = gain1.checked ? false : true;
```

Comment:
use this : const selectedGain = !gain1.checked; for this line and all the similar lines below